src/components/Common/SocialSidebar/Sidebar.stories.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import Sidebar from "./Sidebar";
+
+export default {
+    title: "Components/Sidebar/Sidebar",
+    component: Sidebar,
+};
+
+const Template = (args) => {
+    const [searchTerm, setSearchTerm] = useState("");
+    const [showOnlyFollowing, setShowOnlyFollowing] = useState(false);
+    const [filterLabel, setFilterLabel] = useState("");
+    const [selectedTags, setSelectedTags] = useState([]);
+
+    const handleSearchTermChange = (value) => {
+        setSearchTerm(value);
+    };
+
+    const handleTypeFilter = (filter) => {
+        setFilterLabel(filter);
+    };
+
+    const userFilters = ["Connections", "Following", "Followers"];
+    const exploreFiltersComponent = <div>Explore Filters Component</div>;
+
+    return (
+        <Sidebar
+            {...args}
+            searchTerm={searchTerm}
+            setSearchTerm={setSearchTerm}
```

Comment:
maybe pass the searchTerm as args because it looks strange


https://github.com/thecyberworld/TheCyberHUB/assets/76536506/6621d5aa-c533-44cf-a824-9679c2fec7cb

or disable the option to write inside the input

Reply:
Passed as args